### ISSUE TITLE
Name of UCA in Scriptlinks panel

### ISSUE THREAD

**Issue by johannes-z** [tavikukko/Chrome-SP-Editor#36]
Right now, if you add an UCA with `ScriptBlock` only, it simply says `CustomAction scriptSrc is empty`, which makes it very hard to identify the UCA. A column that showed the name of the UCA would help a lot!

![image](https://user-images.githubusercontent.com/9069888/38017748-3e4eb5f6-3273-11e8-8dfa-fa5429f600bb.png)

**Comment by tavikukko** [comment]
Thanks @johannes-z,
The idea of SP Editor ScriptLinks functionality is to add/remove scriptlinks, not to monitor all UCSs in the SiteCollection. When injecting css, its actually using ScriptBlock, but its parsing the src attribute value from it and displaying that in the editor view. Also not all UCAs have title set, so that would lead to the same problem. So in the case you see this "CustomAction scriptSrc is empty" message, you will know that this UCA is to do something else than injecting files to SharePoint.

**Comment by tavikukko** [comment]
One option would be not to display those which do not inject files to SharePoint, but i find it interesting and usefull to see whats injected and it also gives me possibility to remove that injection, if i know what is it about, afterall, this is designed to be used in development envs only and there you know all your customizations/UCAs.

**Comment by johannes-z** [comment]
@tavikukko True, but as you said, it's also for removing scriptlinks. If there was a name you'd know what you were removing :)

**Comment by tavikukko** [comment]
@johannes-z ok, there is a point So if there is no injection, SP Editor could show the title of the UAC. If title does not exist, "CustomAction scriptSrc is empty" message could be valid?

**Comment by johannes-z** [comment]
I'm not suggesting that it shouldn't show that message if the title is set. I just want to see the title in addition to the scriptlink or that message if that makes sense

**Comment by johannes-z** [comment]
Maybe if there were 2 rows per entry would help? E.g.:

| Sequence | | Remove |
|---:| --- | --- |
| 1 | My CustomAction <br> CustomAction scriptSrc is empty | x |
| 2 | Some other UCA <br> `link to .js file` | x |
